src/components/core/geoChart/geoChartDataLayers.ts

```diff
@@ -108,7 +110,7 @@ export function createPushpinDataLayer(
             [PUSHPIN_STYLE_CIRCLE_SIZE]: createPushpinSizeOptions(geoData, pushpinSizeOptions),
         },
     };
-    if (selectedSegmentItems !== undefined) {
+    if (!isEmpty(selectedSegmentItems)) {
```

Comment:
Is there a bug needs fixed with `isEmpty`? Using plain js comparison is fine to me

Reply:
updated